firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/ImagenGenerationConfig.kt

```diff
@@ -97,3 +117,14 @@ public fun imagenGenerationConfig(
   builder.init()
   return builder.build()
 }
+
+@OptIn(PublicPreviewAPI::class)
+public fun xx() {
+  imagenGenerationConfig {
+    negativePrompt = "People, black and white, painting"
+    numberOfImages = 1
+    aspectRatio = ImagenAspectRatio.SQUARE_1x1
+    imageFormat = ImagenImageFormat.png()
+    addWatermark = false
+  }
+}
```

Comment:
@rlazo uh-oh, was this meant to be merged? It's in the `api.txt` file too!

Reply:
🤦